spec.bs

```diff
@@ -416,22 +416,14 @@ Using its private keys, issuer signs the blinded tokens obtained in the
 specified in the request `Sec-Private-State-Token-Version` header. Encoding the values
 passed in URL `private` and `public` parameters happens in this signing
 step. Issuer returns the signed tokens in the `Sec-Private-State-Token` response header
-value encoded as a base64 byte string. The number of tokens issued is returned
-in `Sec-Private-State-Token-Count` header. Value of this header is the string message of
-
-```
-Issuing <number of tokens> tokens.
-```
-
-with the right nonnegative integer value that specifies the tokens issued.
+value encoded as a base64 byte string.
 
 <div class=example>
 Following snippet displays a typical response demonstrating the Private State Token
```

Comment:
While we're editing, can you say something like "The following..." snippet? example? dunno

Reply:
Done.